interface/interface.dm

```diff
@@ -117,6 +118,7 @@ Hotkey-Mode: (hotkey-mode must be on)
 \tr = throw
 \tm = me
 \tt = say
+\t<B></B>h = talk-wheel

```

Comment:
\th is an escape sequence in BYOND. If anyone can figure out a cleaner way to write this, I'm all ears.